pkg/ctl/create/nodegroup.go

```diff
@@ -42,6 +50,9 @@ func createNodeGroupCmd(g *cmdutils.Grouping) *cobra.Command {
 		fs.StringVar(&cfg.Metadata.Name, "cluster", "", "name of the EKS cluster to add the nodegroup to")
 		cmdutils.AddRegionFlag(fs, p)
 		cmdutils.AddVersionFlag(fs, cfg.Metadata)
+		fs.StringVarP(&clusterConfigFile, "config-file", "f", "", "load configuration from a file")
+		fs.StringVarP(&nodeGroupFilter, "only", "o", "",
+			"select a subset of nodegroups via comma-separted list of globs, e.g.: 'ng-*,nodegroup?,N*group'")
```

Comment:
I think we should consider something other than `-o` as that seems to be commonly used for output, e.g. `-o json`

Reply:
Yes, could be `-O`, but perhaps we could just omit short version for now.

Reply:
K